verticapy/utilities.py

```diff
@@ -1012,8 +1012,17 @@ def isvmap(
             116,
         ):  # 116 is for long varbinary and 115 is for long varchar
             return False
-    except:
-        return False
+    except Exception as e:
+        if "'utf-8' codec can't decode byte" in str(e):
+            try:
+                sql = f"SELECT MAPVERSION({column}) AS isvmap FROM {expr} WHERE {column} IS NOT NULL LIMIT 1;"
```

Comment:
This query is similar to the one in line 1004 except that it doesn't have `, {column}` in the SELECT clause. Couldn't you just drop that from your first query? I don't understand why it is needed there.